_sass/_typography.scss

```
@@ -109,15 +61,63 @@ p {
   margin-bottom: 0.35em;
 }
 
-// Terminal Font helper
-
-.terminal {
-  font-family: 'Terminal Grotesque';
-}
-
 // Bullets
 
 .bullet {
   width: 1em;
   height: 1em;
 }
+
+// Typewriter theme effect
+
+.typing {
+  color: rgba($text-color, 0.8);
+  font-family: $mono-font-family;
+  font-size: $small-font-size;
+  line-height: 1.25em;
+  padding: 1em;
+  margin: 2em 0;
+  width: 75%;
+  background: rgba(0,0,0,.05);
+  height: 18em;
+  overflow: scroll;
```

This gets rid of the x scrollbar and shows the y scrollbar when needed:

```suggestion
  overflow: auto;
```